kong/plugins/prometheus/exporter.lua

```diff
@@ -146,11 +146,12 @@ local upstream_target_addr_health_table = {
   { value = 0, labels = { 0, 0, 0, "dns_error" } },
 }
 
-local function set_healthiness_metrics(table, upstream, target, address, status, metrics_bucket)
+local function set_healthiness_metrics(table, upstream, target, address, status, subsystem, metrics_bucket)
   for i = 1, #table do
     table[i]['labels'][1] = upstream
     table[i]['labels'][2] = target
     table[i]['labels'][3] = address
+    table[i]['labels'][5] = subsystem
```

Comment:
I think we can just set the subsystem label in `upstream_target_addr_health_table` on L142 since
subsystem won't change after module load.

Reply:
Very good catch. Thanks!